rkt/trust.go

```diff
@@ -59,7 +59,15 @@ func init() {
 func runTrust(cmd *cobra.Command, args []string) (exit int) {
 	if flagPrefix == "" && !flagRoot {
 		if len(args) != 0 {
-			stderr.Print("--root required for non-prefixed (root) keys")
+			trustName := args[0]
+			cmdName := "trust"
+			
+			stdout.Printf(`Prevented potentially dangerous unbounded trust of all containers signed by keys from %s
+To continue trusting this root domain:
+	rkt %s --root %s PUBKEY (local key file, no discovery)
+Or, establish a smaller scope with a prefix:
+	rkt %s --prefix "example.com/hello"
```

Comment:
@matthaias that's also my understanding.

I was double checking with the code, and I think the remote keyfile option is there on purpose. Thus I guess doc should just mention `with a path to a key file (no discovery)` (:+1: for spotting it).

Regarding the message, I would try to avoid guessing what the user was doing (ie. `rkt trust my_key.pub.gpg` is a whole different story, but will be shown your error). It is ok if you hardcode `example.com` and keep wording similar to your current message, like:

```
trust: aborting due to implicit unbounded (root) trust
Please provide a specific domain prefix to trust:
    rkt trust --prefix "example.com/foo" [PUBKEY]
    rkt trust --prefix "example.com/foo/*" [PUBKEY]
Otherwise, trust at the root domain (not recommended) must be explicitly requested:
    rkt trust --root PUBKEY
```